### PR TITLE
Update query to set bot_account

### PR DESCRIPTION
**Pull request by deepsidhu85**
## What does this PR do and why?
_Describe in detail what your merge request does and why._

This PR updates the bot_actions and bot_personal_access_tokens concerns to find and set the bot account using the namespace_bot id instead of the user id. The UI and testing have also been updated

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

1. For both projects and groups add a bot account and verify you can delete the bot account
2. Verify tests pass

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [X] I have evaluated the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.

### CODE REVIEW

File: app/views/groups/bots/_bot.html.erb [hunk 1]
```diff
@@ -3,36 +3,36 @@
...
-    <%= bot_account.personal_access_tokens.last.scopes.join(", ") %>
+    <%= bot_account.user.personal_access_tokens.last.scopes.join(", ") %>
   </td>
   <td class="px-6 py-3 whitespace-nowrap">
-    <%= viral_time_ago(original_time: bot_account.created_at) %>
+    <%= viral_time_ago(original_time: bot_account.user.created_at) %>
   </td>
   <td class="px-6 py-3 whitespace-nowrap">
-    <% if bot_account.members.count.positive? %>
+    <% if bot_account.user.members.count.positive? %>
```
Comment (ericenns) [inline app/views/groups/bots/_bot.html.erb:18]: **nitpick (blocking):** This should be changed to get the membership for the `namespace` specified in the `namespace_bot` entry.

File: app/views/projects/bots/_bot.html.erb [hunk 2]
```diff
@@ -3,36 +3,36 @@
...
-    <%= bot_account.personal_access_tokens.last.scopes.join(", ") %>
+    <%= bot_account.user.personal_access_tokens.last.scopes.join(", ") %>
   </td>
   <td class="px-6 py-3 whitespace-nowrap">
-    <%= viral_time_ago(original_time: bot_account.created_at) %>
+    <%= viral_time_ago(original_time: bot_account.user.created_at) %>
   </td>
   <td class="px-6 py-3 whitespace-nowrap">
-    <% if bot_account.members.count.positive? %>
+    <% if bot_account.user.members.count.positive? %>
```
Comment (ericenns) [inline app/views/projects/bots/_bot.html.erb:18]: **nitpick (blocking):** This should be changed to get the membership for the namespace specified in the namespace_bot entry.

**Review by ericenns — CHANGES_REQUESTED** [review]
Looking good, just one thing that I missed in earlier reviews.

File: app/views/groups/bots/_bot.html.erb (hunk 1 above)
Comment (deepsidhu85) [reply app/views/groups/bots/_bot.html.erb]: Right I missed updating this after we created the namespace_bot model. Updated in a37b856b3427dfd7bd949c10e4caa265bf956e47

File: app/views/projects/bots/_bot.html.erb (hunk 2 above)
Comment (deepsidhu85) [reply app/views/projects/bots/_bot.html.erb]: Updated in a37b856b3427dfd7bd949c10e4caa265bf956e47

**Review by ericenns — APPROVED** [review]
Looks good to me! Thanks for making all the changes.

**Pull request merged**